backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Core/ClientInterfaces/ISingleRightClient.cs

```diff
@@ -0,0 +1,20 @@
+using Altinn.AccessManagement.UI.Core.Models.SingleRight.CheckDelegationAccess;
+using Altinn.AccessManagement.UI.Core.Models.SingleRight.CheckDelegationAccess.CheckDelegationAccessDto;
+
+namespace Altinn.AccessManagement.UI.Core.ClientInterfaces
+{
+    /// <summary>
+    ///     Interface for client to make single right requests to access-managment
+    /// </summary>
+    public interface ISingleRightClient
+    {
+        /// <summary>
+        ///     Checks whether the user can delegate the given right to the given party
+        /// </summary>
+        /// <param name="partyId">
+        ///     Used to identify the party the authenticated user is acting on behalf of./param>
```

Comment:
The incomplete XML tag /param> seems to have been copy/pasted a lot, causing a lot of warnings

Reply:
I see that now, they were auto generated. I'll close them